src/vicsetup/defs.mak

```diff
@@ -0,0 +1,16 @@
+TOP=$(abspath $(dir $(word 2, $(MAKEFILE_LIST))))
+
+CC = gcc
+
+CFLAGS += -g
+CFLAGS += -O3
```

Comment:
Debug and optimization should be settable via options.

Reply:
Good idea. Added **DEBUG** and **OPTIMIZE** make switches. Defaults to -03 and no-debug.